node-tests/fixtures/component-test/addon.gts

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'dummy/tests/helpers';
+import { render } from '@ember/test-helpers';
+import Foo from 'dummy/components/foo';
```

Comment:
I'm not sure about this one, I think we may want to import from the addon namespace in this scenario, right? So from `my-addon/components/foo` in the tests?

Reply:
Yes, I think it would be better to use the addon's namespace.